site/js/searchbar.js

```diff
@@ -49,7 +49,7 @@ function toggleActiveClass() {
   }
 }
 
-// toggleHiddenNav adds or removes hidden from nav on mobile
```

Comment:
please use `//` on function comments. And if the code is unnecessary, just remove.